stylesheets/style.css

```diff
@@ -0,0 +1,355 @@
+body {
+  background-color: white;
+}
+
+.date {
+  color: white;
+}
+
+.login {
+  color: white;
+}
+
+header {
+  width: 100%;
+  background-color: red;
+}
+
+footer svg {
+  width: 220px;
+  fill: red;
+}
+
+.logo svg {
+  fill: white;
+}
+
+.user i,
+.bars i {
+  color: lightgray;
+  font-size: 25px;
+}
+
+.grey {
+  color: grey;
+  font-size: medium;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-weight: 900;
+}
+
+.headline-image img {
+  width: 320px;
+  height: 250px;
+}
+
+.headline1 h2 {
+  font-size: large;
+  font-weight: 900;
+  font-family: Arial, Helvetica, sans-serif;
+  width: 320px;
+}
+
+.opinion h2 {
+  width: 270px;
+  font-size: large;
+  font-weight: 900;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.headline-detail {
+  width: 330px;
+  font-weight: 100;
+}
+
+.headline-2 h2 {
+  width: 220px;
+}
+
+.headline-2 img {
+  width: 100px;
+  height: 100px;
+}
+
+.headline-2 p {
+  width: 230px;
+}
+
+.size {
+  border-bottom: 4px solid lightgray;
+  width: 330px;
+}
+
+.debate-img img {
+  width: 70px;
+  height: 70px;
+}
+
+.bp {
+  border-bottom: 1px solid lightgray;
+}
+
+.gray {
+  color: gray;
+}
+
+.thickness {
+  font-weight: 500;
+  width: 300px;
+}
+
+.thick {
+  border: 2px solid lightgray;
+  width: 300px;
+  position: relative;
+}
+
+.debate2 {
+  border-bottom: 2px solid lightgray;
+}
+
+.more-stories img {
+  width: 100px;
+  height: 100px;
+}
+
+.stories h2 {
+  font-size: large;
+  font-weight: 900;
+  font-family: Arial, Helvetica, sans-serif;
+  width: 220px;
+}
+
+.stories p {
+  width: 220px;
+}
+
+.myturn img {
+  width: 320px;
+  height: 250px;
+}
+
+.nursing-homes {
+  border: 1px solid lightgray;
+}
+
+.cover-headline h2 {
+  color: red;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.cover {
+  background-color: red;
+  color: white;
+}
+
+.bolder {
+  font-weight: 700;
+  font-size: small;
+  width: 330px;
+}
+
+.subs-first img {
+  position: relative;
+  width: 220px;
+  margin-left: 50px;
+}
+
+.subs-second img {
+  position: absolute;
+  width: 220px;
+  margin-left: 80px;
+  margin-top: -150px;
+}
+
+.subs-heading {
+  position: absolute;
+  z-index: 1;
+  margin-top: -20px;
+}
+
+.subs-heading h2 {
+  font-size: 12px;
+  font-weight: 700;
+  width: 200px;
+  margin-left: 50px;
+}
+
+.subs-package {
+  font-size: 10px;
+  margin-left: 40px;
+}
+
+.subs-package p {
+  width: 200px;
+}
+
+button {
+  background-color: red;
+  color: white;
+  border: none;
+}
+
+.menu-button button {
+  font-size: 50%;
+  width: 90px;
+  font-weight: 600;
+}
+
+.footer-list {
+  color: grey;
+  list-style: none;
+  font-size: 12px;
+}
+
+.social-icons i {
+  font-size: 32px;
+  color: #808080;
+}
+
+.img-newsletter {
+  width: 320px;
+  height: 150px;
+}
+
+.magazine img {
+  width: 320px;
+}
+
+.editor-pick img {
+  width: 320px;
+}
+
+.digital {
+  font-size: 10px;
+}
+
+.footer-logo svg {
+  width: 180px;
+}
+
+.us img {
```

Comment:
try to make this class `us` descriptive

Reply:
@descholar-ceo Thanks for the code review. I have removed the errors and updated the code.